### DISCUSSION TITLE
project alive?

### DISCUSSION THREAD

**Discussion by rozrabiak** [gioblu/PJON#418] [General]
Hi, is this project still alive?
Nothing happens from long time...

**Comment by fredilarsen** [comment]
Hi @rozrabiak. The author @gioblu and contributors like myself are still alive and well. We have other projects as well (both work, family and other software projects), so I guess time is delegated to what is unfinished, and PJON is in a relatively feature complete state from my point of view. I use it as a basis for my own home automation and it currently does all I need it to do in an extremely stable manner over the years, using my own repo named ModuleInterface on top for standardized messaging and connections to SQL, web pages and MQTT out of the box.

Do you have any specific feature requests or other wishes?

**Reply by gioblu** [reply]
Ciao @rozrabiak yes, you do not see a lot of activity because I am now focused mainly on work, and BIPLAN when I have free time (seldom). I must say that the work on PJON was the most instructive and fun period of my life, and I must thank @fredilarsen for all the support, ideas and elegant source code he contributed over the years. 

If you wish to contribute, we need some help :), the ESP support is flacky, some users forked and developed their own solution, we need some help from an experienced user of those MCUs.

**Reply by rozrabiak** [reply]
> Hi @rozrabiak. The author @gioblu and contributors like myself are still alive and well. We have other projects as well (both work, family and other software projects), so I guess time is delegated to what is unfinished, and PJON is in a relatively feature complete state from my point of view. I use it as a basis for my own home automation and it currently does all I need it to do in an extremely stable manner over the years, using my own repo named ModuleInterface on top for standardized messaging and connections to SQL, web pages and MQTT out of the box.
> 
> Do you have any specific feature requests or other wishes?

My knowledge in programming is very poor - that's the barrier and I can't support that project  .... :(

I need assistance to run communication PJONSoftwareBitBang with stm32f401 and STM32Duino. I try all pins and it's not working. Stm32f4 is not supported by SBB officially.
Is there any solution?

I have this board: https://stm32-base.org/boards/STM32F411CEU6-WeAct-Black-Pill-V2.0.html

Thank you for any answer that will help me ..
